pkg/peertls/tlsopts/options_test.go

```diff
@@ -142,7 +143,7 @@ func TestOptions_ServerOption_Peer_CA_Whitelist(t *testing.T) {
 			dialOption, err := opts.DialOption(target.Id)
 			require.NoError(t, err)
 
-			transportClient := transport.NewClient(opts)
+			transportClient := transport.NewClientWithTimeout(opts, 20*time.Millisecond)
```

Comment:
Is there a reason we are not using default values here?

Reply:
same answer^